examples/compound/go.mod

```diff
@@ -7,7 +7,7 @@ replace github.com/nakji-network/connector/chain/ethereum => ../../chain/ethereu
 replace github.com/nakji-network/connector => ../../
 
 require (
-	github.com/ethereum/go-ethereum v1.10.25
+	github.com/ethereum/go-ethereum v1.10.15
```

Comment:
?

Reply:
Be consistent with the version in `connector` lib